jenkins/bootstrap_test.py

```diff
@@ -1980,6 +1980,8 @@ def testValidJobEnvs(self):
         with open(bootstrap.test_infra('jobs/config.json')) as fp:
             config = json.loads(fp.read())
             for job in config:
+                self.assertTrue('sigOwners' in config[job], job)
+                self.assertTrue(type(config[job]['sigOwners']) == list)
```

Comment:
Please also assert that sigOwners is:
* not empty,
* all elements match a known set of sigs
* include a UNKNOWN sig and seed all the jobs with that.